content/general/concepts/qr-code/qr-code.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'QR Codes'
```

Comment:
@easoftwareGit The metadata needs to look like this:

```
---
Title: 'The Title'
Description: 'A brief description'
Subjects:
  - 'A subject name'
  - 'A second subject name'
  - 'An nth subject name'
Tags:
  - 'A tag'
  - 'A second tag'
  - 'An nth tag'
CatalogContent:
  - 'learn-example-course'
  - 'path/example-path'
---
```

Subjects come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/subjects).
Tags come from [this list](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).
For catalog content use [this document](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md) as a guide.